### DISCUSSION TITLE
Totally useless!

### DISCUSSION THREAD

**Discussion by ObsidianStack** [spatie/ssh#60] [General]
This works only where composer is available, not in environments where composer is not. 
Requiring it as library throws an error

Parse error: syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) on line 11

Which makes this package totally useless for who can't use composer. Clap clap clap.

**Comment by freekmurze** [comment]
Happy new year to you too 🍾

If you 're getting syntax errors, make sure you're using the required PHP version. 

I can definitely recommend using composer for installing PHP package. It's the de facto standard in the PHP community nowadays. Should you decide to pick it up, I'm sure you'll get lots of Clap Clap Clap from the people you work with.

**Comment by WyriHaximus** [comment]
Out of interest, why isn't composer available in your environment, and which environment is that @ITEvo?

**Comment by GrahamCampbell** [comment]
This issue is actually nothing to do with Composer. You are running the code on an EOL version of PHP. You need PHP 7.4 or higher.
